src/styles/calendar.scss

```diff
@@ -0,0 +1,26 @@
+.calendar {
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: start;
+  width: 7 * $date-size + 10px;
+  gap: #{$gap};
+  padding: 20px 10px 0;
+
+  &__date {
+    width: $date-size;
+    height: $date-size;
+    background-color: #eee;
+    box-sizing: border-box;
+    border: 1px solid black;
+    position: relative;
+    transition: transform 500ms;
```

Comment:
would be great to apply the transition to `background-color` too and make a variable for 500ms